server/database.go

```diff
@@ -701,21 +701,24 @@ func (s *Server) composeDatabaseRelationship(ctx context.Context, database *api.
 	}
 
 	rowStatus = api.Normal
-	labels, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
+	labelList, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
 		DatabaseID: &database.ID,
 		RowStatus:  &rowStatus,
 	})
 	if err != nil {
 		return err
 	}
 
-	if len(labels) > 0 {
-		labels, err := json.Marshal(labels)
-		if err != nil {
-			return err
-		}
-		database.Labels = string(labels)
+	labelList = append(labelList, &api.DatabaseLabel{
+		Key:   "environment",
```

Comment:
Need to comment the reason.

Also should we use bb.environment ?

Reply:
I wonder if "environment" is a real label or just a concept mapping to `db.instance.environment.name`?